GUI/MainWindow.xaml.cs

```diff
@@ -69,7 +69,16 @@ public MainWindow()
             FileSpecificParameters.ValidateFileSpecificVariableNames();
 
             // LOAD GUI SETTINGS
-            GuiGlobalParams = Toml.ReadFile<GuiGlobalParams>(Path.Combine(GlobalVariables.DataDir, @"GUIsettings.toml"));
+
+            try
+            {
+                GuiGlobalParams = Toml.ReadFile<GuiGlobalParams>(Path.Combine(GlobalVariables.DataDir, @"GUIsettings.toml"));
+                notificationsTextBox.Document.Blocks.Clear();
```

Comment:
probably not a good idea to clear the notifications on startup, since there can be important warning messages. instead of coding the startup message into the xaml, write it here if the GUISettings.toml doesn't exist